src/example/README.md

```diff
@@ -1,21 +1,20 @@
 # Examples
 
-
+## How to use the examples
 To see the examples in action, you'll need to clone the repository.
 ```bash
 git clone https://github.com/fastjengine/FastJ.git
 ```
 
 Then, run the gradle task `example`. When you do this, you'll need to specify which example you want to run. This can be done by adding `-PtoRun=<example name, without the brackets>`
 ```bash
-./gradlew example -PtoRun=helloworld
+./gradlew example -PtoRun=hellofastj
 ```
 _Having trouble using `gradlew`? Read [this][Terminals Are Different]._
 
+_FastJ requires [Java 11][AdoptOpenJDK-Java11-Link] as a minimum_
```

Comment:
I like this specifically -- it might be moved upwards later on, but I'm realizing I really don't specify the Java requirements anywhere! Thanks for mentioning it.